packages/v4/patternfly-docs/content/design-guidelines/components/skeleton/skeleton.md

```diff
@@ -5,14 +5,22 @@ section: components
 
 ## Usage 
 Skeleton loading should be progressive –– it should load static text first, followed by variable elements of the UI.
```

Comment:
Should put this first sentence at the end of the paragraph instead of at the start